tasks/generate.py

```diff
@@ -29,12 +26,12 @@ def announce(ctx, version):
     contributors_text = '\n'.join('* {}'.format(name) for name in sorted(contributors)) + '\n'
     text = template_text.format(version=version, contributors=contributors_text)
 
-    target = Path(__file__).joinpath('../../doc/en/announce/release-{}.rst'.format(version))
+    target = Path(__file__).parent.joinpath('../doc/en/announce/release-{}.rst'.format(version))
```

Comment:
> I don't see the point in the change at all then, apart from you saying that the first version (i.e. the - in the diff?) doesn't work on Linux. What happens?

Yeah that was the motivation. When I executed this Yesterday it failed with "access denied", changing it as above made it work for me. I didn't thought much at the time about it at the time TBH.